docs/zh/client/java-sdk-user-guide.md

```
@@ -491,6 +509,206 @@ ConfigFile configFile = ConfigService.getConfigFile("test", ConfigFileFormat.XML
+```yaml
+apollo:
+  client:
+    monitor:
+      enabled: true
+      jmx:
+        enabled: true
+```
+
+启动应用后，开启J-console或J-profiler即可查看，这里用J-profiler做例子
+
+![](https://raw.githubusercontent.com/Rawven/image/main/20241020224657.png)
```

Markdownlint

Images should have alternate text (alt text)

(MD045, no-alt-text)